WISE-Community/WISE-Client
# feat(Tag): Add filter units by tag

## Changes
- Added filter units by tag
- Created AbstractTagsMenuComponent that ApplyTagsButtonComponent and TagsFilterComponent both extend

## Test
- Make sure you can filter units in the teacher run list by tag
- Make sure the apply tags menu still works
